Samples/azure-storage/Azure.Ruby/azure_storage/models/storage_account_list_result.rb

```diff
@@ -13,9 +13,6 @@ class StorageAccountListResult
       # their properties.
       attr_accessor :value
 
-      # @return [String]
```

Comment:
This is due to my commit (https://github.com/Azure/autorest/commit/64a7b867315c5e6b23222ca86dab5ca3c7d2580e) which revert my previous commit (https://github.com/Azure/autorest/commit/6b8360de308a91706f08a648cb9256a7f968617c). Somehow my full code regen didn't revert the ruby file.